AshleyLab/motrpac-frontend
# MOTR-155 Changed relative URL references for preview images to absolute

**Key Change**
- Updated relative URLs for preview images to absolute URLs

**How to test using localtunnel:**
- Install Localtunnel : $ npm install -g localtunnel
- Use the command line interface to request a tunnel to your local server: $ lt -p 3000
- You will receive a url, for example https://bitter-goose-76.localtunnel.me/, that you can type it on Facebook to see the preview image.
- For Twitter, previewing only works if you actually tweet the link

**Test Result:**
I tested by hosting with localtunnel and pasting the generated link into the relevant fields for each application. 

- Facebook
<img width="497" alt="Screen Shot 2019-10-10 at 3 56 49 PM" src="https://user-images.githubusercontent.com/29579769/66612787-efe66480-eb77-11e9-8e6b-3d611bea5e32.png">

- Twitter
<img width="592" alt="Screen Shot 2019-10-10 at 3 57 26 PM" src="https://user-images.githubusercontent.com/29579769/66612788-f07efb00-eb77-11e9-921b-8fa9708d0f2f.png">

- Slack
<img width="726" alt="Screen Shot 2019-10-10 at 3 58 10 PM" src="https://user-images.githubusercontent.com/29579769/66612789-f07efb00-eb77-11e9-93ce-e7e6e63b20ff.png">

**Note**
- Twitter only previews the link once tweeted
